mkosi/config.py

```diff
@@ -203,15 +203,15 @@ def parse_path(value: str) -> Path:
     return parse_path
 
 
-def config_make_path_parser(required: bool) -> ConfigParseCallback:
+def config_make_path_parser(required: bool, absolute: bool = True) -> ConfigParseCallback:
     def config_parse_path(dest: str, value: Optional[str], namespace: argparse.Namespace) -> Optional[Path]:
         if dest in namespace:
             return getattr(namespace, dest) # type: ignore
 
         if value and required and not Path(value).exists():
             die(f"{value} does not exist")
 
-        return Path(value).absolute() if value else None
+        return Path(value).absolute() if value and absolute else Path(value) if value else None
```

Comment:
More vertical, but I think easier to parse.
```suggestion
        if value:
            return Path(value).absolute() if absolute else Path(value)
        return None
```